sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -575,12 +575,12 @@ def reset_index(self, level=None, **kwargs):
     if level is not None and not isinstance(level, (tuple, list)):
       level = [level]
     if level is None or len(level) == self._expr.proxy().index.nlevels:
-      # TODO(BEAM-12182): Could do distributed re-index with offsets.
+      # TODO(Issue 20859): Could do distributed re-index with offsets.
       requires_partition_by = partitionings.Singleton(
           reason=(
               f"reset_index(level={level!r}) drops the entire index and "
               "creates a new one, so it cannot currently be parallelized "
-              "(BEAM-12182)."))
+              "(Issue 20859)."))
```

Comment:
I think a URL would be preferable in the error message for easy discovery. Previously BEAM-12182 could likely be found with a Google search, but "20859" will require the user to construct the URL.

Reply:
I ended up just moving everything to URLs. I think the same logic applies more generally than just in error messages.